examples/scripts/making_a_bot.py

```diff
@@ -29,6 +29,7 @@ def run(self) -> None:  # overwrite the AbstractBotJob run method
         comment = 'trim whitespace'
         with gzip.open(self.args.file, 'rb') as fin:
             for row in fin:
+
```

Comment:
Why?

Reply:
There were some code which I added and removed later since that was already taken care of by the `self.save` fn. Removed the extra whitespace now.